sdk/scheduler/src/main/java/com/mesosphere/sdk/offer/DefaultOfferRequirementProvider.java

```diff
@@ -470,7 +470,7 @@ private static boolean isPortResource(Protos.Resource resource) {
         // command and user:
 
         Protos.CommandInfo.Builder commandInfoBuilder = executorInfoBuilder.getCommandBuilder()
-                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib && " +
+                .setValue("export LD_LIBRARY_PATH=$MESOS_SANDBOX/libmesos-bundle/lib:$LD_LIBRARY_PATH && " +
```

Comment:
Not for this PR, but should we consider making the executor command prefix configurable with a reasonable default? Most people won't need or want to touch it, but those that do would probably appreciate not being forced into using our bundle.